manifest.json

```diff
@@ -36,12 +36,19 @@
           "label": "link_color_label",
           "value": "#1F73B7"
         },
+        {
+          "identifier": "hover_link_color",
+          "type": "color",
+          "description": "hover_link_color_description",
+          "label": "hover_link_color_label",
```

Comment:
We don't have these strings.
If we're adding a new setting it needs to be localized first.

Reply:
Isn't this only used for SCSS? I don't understand what needs localization?

Reply:
This variables are exposed as settings in the UI and need to be localized:

<img width="920" alt="Screenshot 2022-02-09 at 14 35 10" src="https://user-images.githubusercontent.com/1159544/153211990-8782375f-ebe8-440c-b113-4f16c0f8b6a4.png">

They're not just available to CSS. Please see [documentation](https://github.com/zendesk/copenhagen_theme#manifest-file) for more info.